apps/ledger-live-mobile/e2e/jest.config.js

```diff
@@ -21,7 +21,7 @@ const jestAllure2ReporterOptions = {
 
 module.exports = async () => ({
   rootDir: "..",
-  maxWorkers: process.env.CI ? 2 : 1,
+  maxWorkers: "50%",
```

Comment:
This reduces emulator/simulator number to one on the CI. Are we sure we want this ?
Could also be the reason why test are more stable (less ressources used with only one device)

Reply:
Ah, interesting, I wonder why. So would you prefer `process.env.CI ? "50%" : 1,` ?